ckanapi/remoteckan.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, address, apikey=None, user_agent=None, get_only=False):
             self.parallel_limit = PARALLEL_LIMIT
 
     def call_action(self, action, data_dict=None, context=None, apikey=None,
-            files=None):
+            files=None, **kwargs):
```

Comment:
Just do a `requests_kwargs = requests_kwargs or {}` in the body. Mutable default arguments freak me out